### PR TITLE
added validation and functionality to contact page

### PR DESCRIPTION
**Pull request by ananyag309**
### Issue #320 

added validation and functionality to contact page - Added validation and improved functionality to the contact page, ensuring that all required fields are properly validated before form submission. This update enhances the user experience by providing real-time feedback and ensuring data integrity with proper display of confirmation message.

working demo :

Uploading WhatsApp Video 2024-10-13 at 16.57.44_90f9b37e.mp4…

## Type of PR
- **Add X in the box to specify the improvement type.**
- [X] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): ___________  

## Description
This pull request aims to [briefly describe what the pull request does, e.g., "add a new feature," "fix a bug," "improve documentation," etc.]. 

## Screenshots / Videos (if applicable)
**Before:**
- [Describe the state before the changes, e.g., "The section headings in the footer were misaligned, leading to an inconsistent and cluttered appearance."]

**After:**
- [Describe the state after the changes, e.g., "The section headings in the footer are now properly aligned, providing a clean and professional look."]

## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

### Additional Context
[Provide any additional context about the changes, such as specific challenges you faced or decisions you made. This can help reviewers understand the rationale behind your changes.]

Thank you for reviewing my pull request!

### CODE REVIEW

**Comment by ananyag309** [comment]
@swaraj-das please review this pr and merge it

File: index.html [hunk 1]
```diff
@@ -557,20 +566,50 @@ <h2>Contact Us</h2>
...
+        chatbotId="awMKW3zFwW7jzUNsN4Q8O"
         domain="www.chatbase.co"
         defer>
         </script>
         <script src="https://unpkg.com/aos@2.3.1/dist/aos.js"></script>
         <script>
             AOS.init();
         </script>
+
+    <script>
```
Comment (swaraj-das) [inline index.html:584]: Add this code in js file.

**Comment by ananyag309** [comment]
@swaraj-das done

**Comment by ananyag309** [comment]
@mastansayyad please have a look into it as well it's been 2 weeks but my pr is not yet merged

**Review by MastanSayyad — APPROVED** [review]
approved

**Pull request closed without merging**